### DISCUSSION TITLE
Other board than Teensy 2.0

### DISCUSSION THREAD

**Discussion by TheProfessorBE** [tloimu/adapt-ffb-joy#43] [General]
Does anyone have a suggestion for a different board than a Teensy 2.0? I have trouble sourcing one locally here. I want to reinvigorate my love for my MS FFB Pro, and be nostalgic again :)

**Comment by Hedda** [comment]
If can code then been suggested to modify the source code so it can work on "Arduino Pro Micro" or "Arduino Leonardo" and their clones. 

See -> https://github.com/tloimu/adapt-ffb-joy/issues/20 (and https://github.com/tloimu/adapt-ffb-joy/discussions/38)

Not sure if possible to get working without changing the source code, but also check out some forks, like example by JayBee and Kreeblah:

https://github.com/JayBee-git/adapt-ffb-joy/

https://github.com/Kreeblah/SidewinderToUSBV2

A quick search digging at least made it seem to me that JayBee's fork at https://github.com/JayBee-git/adapt-ffb-joy/ should contain the latest code of Adapt-FFB-Joy as well as some consolidation fixes and additional features from other forks of adapt-ffb-joy, and it in also have more branches with additional features. An [old community discussion](https://www.descentbb.net/viewtopic.php?t=19061&start=280) mentions that JayBee's fork included changes from https://github.com/DEV-OGRE/adapt-ffb-joy/tree/feature/shift-and-frame-buffer as well as https://github.com/r8dhex/adapt-ffb-joy/tree/feature/shiftEnable and https://github.com/r8dhex/adapt-ffb-joy/tree/feature/moreButtons) too, I think?

According to the wiki at https://github.com/tloimu/adapt-ffb-joy/wiki there was also an old project making it into an Arduino library:

https://code.google.com/archive/p/sidewinder-arduino/

PS: Also noticed NolanNicholson started a other project based on Arduino Pro Micro -> https://github.com/NolanNicholson/SWFFB-Arduino

**Reply by ej113** [reply]
See my comment in #20
